smartlearner/interfaces/model.py

```diff
@@ -30,8 +22,15 @@ def parameters(self):
 
     @abstractmethod
     def save(self, path):
+        """ Saves model information to disk. """
         raise NotImplementedError("Subclass of 'Model' must implement 'save(path)'.")
 
-    @abstractclassmethod
+    @abstractmethod
     def load(self, path):
+        """ Loads model information from disk. """
         raise NotImplementedError("Subclass of 'Model' must implement 'load(path)'.")
+
+    @classmethod
+    def create(cls, path):
```

Comment:
What about if you want to use a trained model afterwards? The create method is useful.